libkbfs/identify_util.go

```diff
@@ -5,68 +5,139 @@
 package libkbfs
 
 import (
+	"errors"
 	"fmt"
 
 	"github.com/keybase/client/go/libkb"
 	"github.com/keybase/client/go/protocol/keybase1"
 	"golang.org/x/net/context"
 )
 
-func identifyUID(ctx context.Context, nug normalizedUsernameGetter, identifier identifier, uid keybase1.UID, isPublic bool) error {
+func identifyUID(ctx context.Context, nug normalizedUsernameGetter,
```

Comment:
Also, this is a small style nit, but I try to organize functions so that within a file they only call functions that are above them in the file.  Not a style requirement, but you'd make me happy if you reorganized these functions that way!